index.js

```diff
@@ -14,7 +14,15 @@ module.exports = function(content) {
 		content: content,
 		regExp: query.regExp
 	});
+	var path = query.path;
```

Comment:
I would use different name to avoid confusion with "[path]" placeholder - e.g. "base" or "publicPath"